### PR TITLE
Mike/2022 updates

### PR DESCRIPTION
**Pull request by ghost**
- Rewritten footer
- Adjusted naming in navbar
- Global breakpoint of 900px
- Depreciating connect page
- Updated board member list
- Updated board member photos
- Adjusted wording of front page components
- Implemented 404 page graphics

### CODE REVIEW

File: src/lib/components/sections/navbar.svelte [hunk 1]
```diff
@@ -108,19 +108,21 @@
...
+  /* nav .page-nodebuds {
     font-weight: 700;
   }
 
   nav .page-nodebuds:hover,
   nav .page-nodebuds[aria-current='true'] {
     color: var(--acm-red);
   }
-
+ */
```
Comment (EthanThatOneKid) [inline src/lib/components/sections/navbar.svelte]: Nit: unused comment in CSS.

File: src/routes/about.svelte [hunk 2]
```diff
@@ -3,7 +3,7 @@
   import OfficerProfileList from '$lib/components/about/officer-profile-list.svelte';
   import Spacing from '$lib/components/sections/spacing.svelte';
   import Header from '$lib/components/about/what-is-acm.svelte';
-  import Admonition from '$lib/components/utils/admonition.svelte';
+  /* import Admonition from '$lib/components/utils/admonition.svelte'; */
```
Comment (EthanThatOneKid) [inline src/routes/about.svelte]: Unused comment.

File: src/lib/components/sections/navbar.svelte [hunk 3]
```diff
@@ -5,14 +5,14 @@
...
+    { title: 'About', path: '/about' },
+    { title: 'Events', path: '/events' },
+    { title: 'Paths', path: '/paths' },
     {
-      title: '<span class="headers">node</span>Buds',
+      title: 'Mentorship',
       path: '/nodebuds',
     },
-    { title: 'connect', path: '/connect' },
+    /* { title: 'connect', path: '/connect' }, */
```
Comment (EthanThatOneKid) [inline src/lib/components/sections/navbar.svelte]: The connect page seems to have been replaced by the updated footer. Nice!

File: src/routes/__error.svelte [hunk 4]
```diff
@@ -1,53 +1,51 @@
 <script>
   import { dev } from '$app/env';
```
Comment (EthanThatOneKid) [inline src/routes/__error.svelte]: Since this is never used, feel free to delete this import.

File: src/routes/__error.svelte [hunk 5]
```diff
@@ -1,53 +1,51 @@
...
   export let status = 404;
-  export let error = new Error('Page Not Found');
+  /* export let error = new Error('Page Not Found'); */
```
Comment (EthanThatOneKid) [inline src/routes/__error.svelte]: Nit: unused comment.

File: src/routes/__error.svelte [hunk 6]
```diff
@@ -1,53 +1,51 @@
...
+
+<!-- <h1>{status} {error.message}</h1>
 <div class="image-container">
   <img src="/assets/png/acm-shark.png" alt="ACM mascot with 404 error" />
 </div>
 {#if dev && error.stack}
   <pre><code>{error.stack}</code></pre>
-{/if}
-
+{/if} -->
```
Comment (EthanThatOneKid) [inline src/routes/__error.svelte]: Feel free to remove any unused comments.

**Review by EthanThatOneKid — CHANGES_REQUESTED** [review]
Literally just a few nitpicks from me mostly to remove any comments that we don't need anymore. The new footer, finalized 404 page, and other improvements look great!

File: src/routes/__error.svelte (hunk 6 above)
Comment (ghost) [reply src/routes/__error.svelte]: Fixed

File: src/routes/__error.svelte (hunk 5 above)
Comment (ghost) [reply src/routes/__error.svelte]: Fixed

File: src/routes/__error.svelte (hunk 4 above)
Comment (ghost) [reply src/routes/__error.svelte]: Fixed

File: src/lib/components/sections/navbar.svelte (hunk 3 above)
Comment (ghost) [reply src/lib/components/sections/navbar.svelte]: Fixed

File: src/routes/about.svelte (hunk 2 above)
Comment (ghost) [reply src/routes/about.svelte]: Fixed

File: src/lib/components/sections/navbar.svelte (hunk 1 above)
Comment (ghost) [reply src/lib/components/sections/navbar.svelte]: Fixed

**Review by EthanThatOneKid — APPROVED** [review]
Looks great! Thanks for the contributions!

**Pull request merged**